index.html

```diff
@@ -305,9 +305,10 @@ <h4>Computation steps</h4>
                 </pre>
               </details></div>
             </li>
-            <li id="step2C">Otherwise, if the <code>current node</code> is a control embedded within the label (e.g. the <code>label</code> element in HTML or any element directly referenced by <code>aria-labelledby</code>) for another <a class="termref">widget</a>, where the user can adjust the embedded control's value, then return the embedded control as part of the text alternative in the following manner:
+            <li id="step2C">Otherwise, if the <code>current node</code> is an embedded control whose value can be adjusted by the user, and the <code>current node</code> is a descendant of an element whose <a class="termref internalDFN" href="#dfn-accessible-name" data-link-type="dfn">Accessible Name</a> or <a class="termref internalDFN" href="#dfn-accessible-description" data-link-type="dfn">Accessible Description</a> is being computed, then include the embedded control as part of the text alternative in the following manner:
```

Comment:
adding `whose value can be adjusted` would remove readonly controls from the embedded control portion of the accname algo... I think it should be removed.